pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -192,3 +212,21 @@ func newCDIForCR(cr *hcov1alpha1.HyperConverged) *cdi.CDI {
 		},
 	}
 }
+
+// newNetworkAddonsForCR returns a NetworkAddonsConfig CR
+func newNetworkAddonsForCR(cr *hcov1alpha1.HyperConverged) *networkaddons.NetworkAddonsConfig {
+	labels := map[string]string{
+		"app": cr.Name,
+	}
+	return &networkaddons.NetworkAddonsConfig{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      networkaddonsnames.OPERATOR_CONFIG,
+			Namespace: networkaddonsnames.APPLIED_NAMESPACE,
```

Comment:
The hyperconverged-cluster-operator is not (or at least should not) have permissions to be creating components across namespaces. Maybe I don't understand the direction that we are going but I would expect _User subscribes to the HCO and gets the HCO operator and the component operators (KubeVirt, CDI, ClusterNetworkAddons, etc.). Then a HCO CR would result in all of the components installed in the namespace where the CR is created._

Reply:
@rthallisey you may be most qualified to direct here.

Reply:
Sorry this is a mistake, NetworkAddonsConfig is cluster-wide, non-namespaced resource. Will fix it.

There are two issues connected to your request:

1) Since network addons alter hosts, they can be deployed only once, therefore we have the resource cluster-wide and with a single acknowledged name "cluster". IIUIC, this is the same for CDI, therefore we could copy CDI leader election.
2) Network addons operator deploys third-party components (multus and linux bridge CNI) into their own namespaces since they are self-container and separated projects.